test/functional/p2p_segwit.py

```diff
@@ -3,17 +3,82 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test segwit transactions and blocks on P2P network."""
+from binascii import hexlify
+import math
+import random
+import struct
+import time
 
-from test_framework.mininode import *
-from test_framework.test_framework import BitcoinTestFramework
-from test_framework.util import *
-from test_framework.script import *
 from test_framework.blocktools import create_block, create_coinbase, add_witness_commitment, get_witness_script, WITNESS_COMMITMENT_HEADER
 from test_framework.key import CECKey, CPubKey
-import math
-import time
-import random
-from binascii import hexlify
+from test_framework.mininode import (
+    BIP125_SEQUENCE_NUMBER,
+    CBlock,
+    CBlockHeader,
+    CInv,
+    COutPoint,
+    CTransaction,
+    CTxIn,
+    CTxInWitness,
+    CTxOut,
+    CTxWitness,
```

Comment:
Those are message types and not from mininode. I'd prefer to keep it the way it was over changing it in the wrong way.

Reply:
Thanks for pointing this out. These were previously being imported indirectly through the `from mininode import *` import, but you're right that I should fix that to import from messages.py in this commit.

I've now put them in the right place.